src/client/main.cpp

```diff
@@ -52,6 +52,28 @@ class ProtocachedClient {
     return result;
   }
 
+  ResponseResult Set(const ::std::string &key, const ::std::string &value) {
+    ::protocached::SetRequest request;
+    request.set_key(key);
+    CachedValue cached_value;
+    cached_value.set_value(value);
```

Comment:
These are what I was talking about, and should be done before calling the client.